[DOC-812] CDC refresh interval (#28217)
yugabyte/yugabyte-db: docs/content/preview/explore/change-data-capture.md

  --tserver_flags="cdcsdk_publication_list_refresh_interval_secs=120"
```

> When you create a table, it is added to the replication slot after the publication list is refreshed. This is done at an interval specified using the cdcsdk_publication_list_refresh_interval_secs flag. Any data that is written between table creation and when the table is added to the publication list is not be delivered as part of the publication. By default, the interval is 15 minutes. Setting the flag to 2 minutes (120 seconds) reduces this delay. For more information, refer to [YugabyteDB semantics](../../develop/change-data-capture/using-logical-replication/advanced-topic/#yugabytedb-semantics).

Reviewer: ```suggestion
When you create a table, it is added to the replication slot after its publication list is refreshed, which occurs at an an interval specified by the `cdcsdk_publication_list_refresh_interval_secs` flag. Any data that is written between table creation and when the table is added to the publication list is not delivered as part of the publication. The default refresh interval is 15 minutes, but you can reduce this delay by setting the flag to 2 minutes (120 seconds). For more information, refer to [YugabyteDB semantics](../../develop/change-data-capture/using-logical-replication/advanced-topic/#yugabytedb-semantics).
```
